docs/testing.md

```diff
@@ -0,0 +1,68 @@
+# Testing
+
+We want to maintain a project with a high coverage (aiming for 100%). Our main [coverage criterion](https://en.wikipedia.org/wiki/Code_coverage#Coverage_criteria) is **branch coverage**, that is making sure all possible code branches are covered by test. [Coveralls](https://coveralls.io/github/mozilla/addons-frontend) helps us monitor the status of our test suite coverage and it is a good idea to double-check its output when you are opening a Pull Request. You can also [generate a coverage report locally](https://github.com/mozilla/addons-frontend/#code-coverage).
+
+## General guidelines
+
+- Imports should be alphabetized, even in test file.
+- Comments should be full sentences to ease readability.
+- There are a lot of helpers in the `tests/unit/helpers` and `tests/unit/amo/helpers` modules.
+- Use the action creators instead of hard coding props. This applies to both UI components and reducers/sagas.
+- Try to use constants (likely from `core/constants`) as much as you can. Corollary: avoid hard coded values.
```

Comment:
I worry about the word "corollary" for ESL folks 😅

"As much as you can" is a bit of a weird guideline too. Maybe this could be:

```markdown
Use constants (see `core/constants`) when using the same value across files. This avoids hard-coding values and [magic constants](https://en.wikipedia.org/wiki/Magic_constant).
```

Reply:
Sounds good to me